packages/@orchidui-vue/src/Form/Calendar/OcCalendar.vue

```diff
@@ -246,7 +246,10 @@ const isDayDisabled = (day) => {
       (props.minDate &&
         currentDate.getTime() < new Date(props.minDate).getTime()) ||
       (props.maxDate &&
-        currentDate.getTime() > new Date(props.maxDate).getTime())
+        // TODO: improve this piece of logic
+        // props.maxDate is always lower than currentDate by few seconds. Depending on browser's speed (CPU). Why? maxDate is defined before currentDate, so it has lower "Time" than currentDate
+        // Added 60 seconds. To not disable maxDate (on page load).
+        currentDate.getTime() > new Date(props.maxDate).getTime() + 60000)
```

Comment:
Try to use Day.js and set hours, minutes and sec to 0

Reply:
This file doesn't include day.js, so I thought I will stick with native Date object
I'll have to revert all usages of Date object into day.js
Is it fine to have Date and Dayjs objects in the same page? 
cc @kewcoder @mykhailo-diakovych

Reply:
To avoid adding seconds and avoid problems with different initialization we could cover all dates in "dayjs()" with format example 2024-01-25 and we would not be affected by hours, min, sec. WDYT? @FaresG @kewcoder

Reply:
Also, dayjs() will make the code look more appropriate

Reply:
> I can include day.js on a different PR. WDYT @mykhailo-diakovych ?

Okay)

Reply:
thank you so much @FaresG @mykhailo-diakovych @kutsaniuk

Reply:
when will you be able to add the pr with dayjs @FaresG ?

Reply:
> when will you be able to add the pr with dayjs @FaresG ?

I will notify you when I create a PR